apps/vscode-extension/src/commands/generate/fixture.command.ts

```diff
@@ -25,7 +25,6 @@ const isHTMLTemplateDocument = (doc: vscode.TextDocument | undefined) => {
 /**
  * Generate component command
  * @param _context
- * @returns
```

Comment:
I am a bit surprised that this is a manual fix and not an automatic one.

Reply:
because there is a rule that ask you to put a description if you put the `@returns`